test/files/run/t8549.scala

```diff
@@ -155,8 +162,7 @@ object Test extends App {
   check(immutable.Queue())(         "rO0ABXNyACBzY2FsYS5jb2xsZWN0aW9uLmltbXV0YWJsZS5RdWV1ZZY146W3qSuhAgACTAACaW50ACFMc2NhbGEvY29sbGVjdGlvbi9pbW11dGFibGUvTGlzdDtMAANvdXRxAH4AAXhwc3IAMnNjYWxhLmNvbGxlY3Rpb24uaW1tdXRhYmxlLkxpc3QkU2VyaWFsaXphdGlvblByb3h5AAAAAAAAAAEDAAB4cHNyACxzY2FsYS5jb2xsZWN0aW9uLmltbXV0YWJsZS5MaXN0U2VyaWFsaXplRW5kJIpcY1v3UwttAgAAeHB4cQB+AAQ=")
   check(immutable.Queue(1, 2, 3))(  "rO0ABXNyACBzY2FsYS5jb2xsZWN0aW9uLmltbXV0YWJsZS5RdWV1ZZY146W3qSuhAgACTAACaW50ACFMc2NhbGEvY29sbGVjdGlvbi9pbW11dGFibGUvTGlzdDtMAANvdXRxAH4AAXhwc3IAMnNjYWxhLmNvbGxlY3Rpb24uaW1tdXRhYmxlLkxpc3QkU2VyaWFsaXphdGlvblByb3h5AAAAAAAAAAEDAAB4cHNyACxzY2FsYS5jb2xsZWN0aW9uLmltbXV0YWJsZS5MaXN0U2VyaWFsaXplRW5kJIpcY1v3UwttAgAAeHB4c3EAfgADc3IAEWphdmEubGFuZy5JbnRlZ2VyEuKgpPeBhzgCAAFJAAV2YWx1ZXhyABBqYXZhLmxhbmcuTnVtYmVyhqyVHQuU4IsCAAB4cAAAAAFzcQB+AAgAAAACc3EAfgAIAAAAA3EAfgAGeA==")
 
-  // TODO scala/bug#8576 throws scala.UnitializedFieldError under -Xcheckinit
-  // check(new immutable.Range(0, 1, 1))( "rO0ABXNyACBzY2FsYS5jb2xsZWN0aW9uLmltbXV0YWJsZS5SYW5nZWm7o1SrFTINAgAHSQADZW5kWgAHaXNFbXB0eUkAC2xhc3RFbGVtZW50SQAQbnVtUmFuZ2VFbGVtZW50c0kABXN0YXJ0SQAEc3RlcEkAD3Rlcm1pbmFsRWxlbWVudHhwAAAAAQAAAAAAAAAAAQAAAAAAAAABAAAAAQ==")
+  check(new immutable.Range(0, 1, 1))( "rO0ABXNyACBzY2FsYS5jb2xsZWN0aW9uLmltbXV0YWJsZS5SYW5nZWm7o1SrFTINAgAGSQADZW5kWgAHaXNFbXB0eUkALXNjYWxhJGNvbGxlY3Rpb24kaW1tdXRhYmxlJFJhbmdlJCRsYXN0RWxlbWVudEkAMnNjYWxhJGNvbGxlY3Rpb24kaW1tdXRhYmxlJFJhbmdlJCRudW1SYW5nZUVsZW1lbnRzSQAFc3RhcnRJAARzdGVweHAAAAABAAAAAAAAAAABAAAAAAAAAAE=")
```

Comment:
I reset the expected values to what 2.12.10 produces, e.g. 

```
scala>  import java.util.Base64;  import scala.reflect.io.File; def printBase64Binary(x: Array[Byte]): String = Base64.getEncoder().encodeToString(x); def parseBase64Binary(x: String): Array[Byte] = Base64.getDecoder().decode(x)
import java.util.Base64
import scala.reflect.io.File
printBase64Binary: (x: Array[Byte])String
parseBase64Binary: (x: String)Array[Byte]

scala>   def serialize(o: AnyRef): String = {
     |     val bos = new java.io.ByteArrayOutputStream()
     |     val out = new java.io.ObjectOutputStream(bos)
     |     out.writeObject(o)
     |     out.flush()
     |     printBase64Binary(bos.toByteArray())
     |   }
serialize: (o: AnyRef)String


scala> println(serialize(new immutable.Range(0, 1, 1)))
rO0ABXNyACBzY2FsYS5jb2xsZWN0aW9uLmltbXV0YWJsZS5SYW5nZWm7o1SrFTINAgAGSQADZW5kWgAHaXNFbXB0eUkALXNjYWxhJGNvbGxlY3Rpb24kaW1tdXRhYmxlJFJhbmdlJCRsYXN0RWxlbWVudEkAMnNjYWxhJGNvbGxlY3Rpb24kaW1tdXRhYmxlJFJhbmdlJCRudW1SYW5nZUVsZW1lbnRzSQAFc3RhcnRJAARzdGVweHAAAAABAAAAAAAAAAABAAAAAAAAAAE=
```

These commented out are stale in some cases, the disabled tests date back to 2014 #3739.